config/advanced-install/namespaced-controller-wo-crds.yaml

```diff
@@ -136,6 +136,8 @@ data:
             maxMsgs: 2000000
             maxAge: 168h
             maxBytes: -1
+            # we use file storage by default, uncomment below to use in memory storage for stream configuration
```

Comment:
Use the one from JetSteam for configuration (0: File, 1: Memory), and give comments(like the `retention` above).